models/classification/efficient_net_wrapper.py

```diff
@@ -0,0 +1,10 @@
+from efficientnet_pytorch import EfficientNet
+import utils.logging
+
+"""Wrapper on top of efficientnet model implementation
+"""
+class EfficientNetWrapper:
+    def __init__(self, model_name:str):
+        self.model = EfficientNet.from_pretrained(model_name)
```

Comment:
It could be a "create" method and directly return the instance of the model

Reply:
or at this point the factory could create it directly haha...

Reply:
Indeed factory shoulkd just return it no need for the wrapper